core/src/test/java/org/elasticsearch/transport/netty/NettyTransportIT.java

```diff
@@ -140,6 +142,8 @@ protected String handleRequest(Channel channel, StreamInput buffer, long request
                             if (reg == null) {
                                 throw new ActionNotFoundTransportException(action);
                             }
+                            transport.inFlightRequestsBreaker.addEstimateBytesAndMaybeBreak(messageLengthBytes, "Node in-flight requests");
```

Comment:
if we push channel creation and circuit breaking to the parent, we can remove this here.